### DISCUSSION TITLE
How to use global @types/@types/google.maps installed from package.js

### DISCUSSION THREAD

**Discussion by scrtechnologies** [nuxt/nuxt#15885] [Questions]
Hello. I use library which uses google maps api. I installed @types/google.maps for that library but Nuxt still doesn't recognise these types.

Is there a way how to use typescript types from NPM in Nuxt 3?

Thank you

**Comment by ndragun92** [comment]
Have u added types in `tsconfig.json` config?

```
{
  ...,
  "compilerOptions": {
    "types": [
      "@types/google.maps"
    ]
  },
}
```
